pincer/client.py

```diff
@@ -857,5 +860,139 @@ async def get_webhook(
         """
         return await Webhook.from_id(self, id, token)
 
+    async def get_current_user(self) -> User:
+        """|coro|
+        The user object of the requester's account.
+
+        For OAuth2, this requires the ``identify`` scope,
+        which will return the object *without* an email,
+        and optionally the ``email`` scope,
+        which returns the object *with* an email.
+
+        Returns
+        -------
+        :class:`~pincer.objects.user.user.User`
+        """
+        return User.from_dict(
+            construct_client_dict(self, self.http.get("users/@me"))
+        )
+
+    async def modify_current_user(
+        self, username: Optional[str] = None, avatar: Optional = None
+    ) -> User:
```

Comment:
`avatar` should be marked as TODO I guess?

Reply:
I think its possible to implement now. Check my questionable guide on page two of the endpoints spreadsheet.